src/cloudWatchLogs/commands/searchLogGroup.ts

```diff
@@ -70,6 +70,14 @@ function handleWizardResponse(response: SearchLogGroupWizardResponse, registry:
     return initialStreamData
 }
 
+export function clearStreamIdMapOnDocumentClose(registry: LogStreamRegistry, doc: vscode.TextDocument): void {
+    vscode.workspace.onDidCloseTextDocument((document: vscode.TextDocument) => {
```

Comment:
this creates a handler. If there's no way to force vscode to trigger the handler, you may need to create a function from the `(document: vscode.TextDocument) => { ... }` closure here.